src/file/file.service.ts

```diff
@@ -253,11 +251,9 @@ export class FileService {
    * @param ownerID - owner of said folder.
    * @param query - the conditions.
    */
-  private static async getPopulatedChildren(folderID: string, ownerID: string, query: Partial<IFile>) : Promise<ResFile> {
-    const ancestor: ResFile = new ResFile(await this.getById(folderID));
+  private static async getPopulatedChildren(folderID: string, ownerID: string, query: Partial<IFile>) : Promise<ResFile[]> {
```

Comment:
This function can be removed and have it's name replacing the `_recursive` function name.

Reply:
done

Reply:
done